data/containers/ibm_hpvs/bci-busybox

```diff
@@ -0,0 +1,6 @@
+services:
+  busybox-nc-test:
+    image: registry.suse.com/bci/IMG@DGST
+    ports:
+      - "8222:5000"
+    command: ["nc", "-n", "-v", "-l", "-p5000", "-e", "/bin/sh"]
```

Comment:
I know that the workload runs ideally only several seconds. But I'd still change this to something more secure.
```suggestion
    command: ["nc", "-n", "-v", "-l", "-p5000", "-e", "/bin/echo", "Have a lot of fun"]
```